examples/js/loaders/ColladaLoader.js

```diff
@@ -3183,14 +3183,26 @@ THREE.ColladaLoader.prototype = {
 
 		}
 
+		var fallbackMaterial = new THREE.MeshBasicMaterial( { color: 0xff00ff ) };
```

Comment:
There appears to be a syntax error here!  ( { color: 0xff00ff ) }

Reply:
Good catch! Can you make a PR with the change?

Reply:
Yes, sure thing, I should have mentioned it before! ha
https://github.com/mrdoob/three.js/pull/14464